packages/medusa/src/loaders/index.ts

```diff
@@ -164,6 +163,8 @@ export default async ({
     expressApp,
     rootDirectory
   )
+
+  const { createDefaultsWorkflow } = await import("@medusajs/core-flows")
```

Comment:
Importing `core-flows` package has a side-effect that it registers workflows and steps as soon as the package is imported.

Therefore, the instrumentation code is not able to apply tracing to core flows, since they are registered before the instrumentation code triggers.

Reply:
this should be fixed with out last changes 👍